docs/self-managed/platform-deployment/amazon-eks.md

```diff
@@ -0,0 +1,47 @@
+---
+id: amazon-eks
+title: "Amazon EKS"
+---
+
+Amazon Elastic Kubernetes Service ([Amazon EKS](https://aws.amazon.com/eks/)) is a managed
+container service to run and scale Kubernetes applications in the cloud or on-premises.
```

Comment:
instead of on-premises, do you mean Self-Managed here?

Reply:
nope, here I'm referring to Amazon EKS functionally, not Camunda Platform. (it's actually how Amazon EKS is described on its [home page](https://aws.amazon.com/eks/)).